voting_ml/example_ftsel.py

```diff
@@ -1,21 +1,65 @@
 import feature_selection
+from sklearn.model_selection import train_test_split
 
 def main():
 
-    ftsel = feature_selection.FeatureSelection(necess_que_file="../extern/manage_data/list_all_questions.txt", unnecess_que_file="../extern/manage_data/list_unnecessary_columns.txt", bool_necess_que=False, run_name="test")
-
-    dataset = ftsel.get_ftsel_original_data()
     
-    data_ftsel, ftsel_questions = ftsel.ftsel_chi2(dataset,KBest = 20)
-
-    print("data_ftsel",data_ftsel)
-    print("ftsel_questions",ftsel_questions)
-
-    df = ftsel.ft_corr(data_ftsel, ftsel_questions)
-
+    #using chi2 method to select features
     
+    chi2_ftsel = feature_selection.FeatureSelection(necess_que_file="../extern/manage_data/list_all_questions.txt", unnecess_que_file="../extern/manage_data/list_unnecessary_columns.txt", bool_necess_que=False, run_name="test_chi2")
+    chi2_dataset = chi2_ftsel.get_ftsel_original_data()
+    X = chi2_dataset[:, :-1]
+    y = chi2_dataset[:,-1]
+    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.33, random_state=1)
```

Comment:
I know this is just an example, but is there any particular reason the test_size is 0.33 here?

Reply:
Just an example. Followed from a webpage.